README.md

```diff
@@ -61,10 +61,49 @@ note is that the endpoint for submitting spot requests requires a fully formed
 and valid `LaunchSpecification`.
 
 ## Hacking
+
 ```
 git clone https://github.com/taskcluster/ec2-manager
 cd ec2-manager
-yarn
+yarn install --frozen-lockfile --ignore-optional
+yarn test
+```
+
+Tests require a working PostgreSQL server initialized with the
+schema of this project.
+
+To use Docker to start a PostgreSQL server:
+
+```
+docker run --rm -p 5432:5432 postgres:9.6
+```
+
+This will pull down and run a PostgreSQL server in the local terminal.
+It will map port 5432 in the container (the PostgreSQL server) to port
+5432 on the Docker host. The container will run in the foreground and take
+over the terminal. You can also use `docker run -d` to detach from the
+current terminal.
+
+Once the PostgreSQL server is running, create database schemas
+necessary to run the tests:
+
+```
+psql -h localhost -p 5432 -U postgres -f sql/create-db.sql
```

Comment:
Running the tests will also wipe out a development database, this command is useful to include because it shows how to recreate the DB for local dev usage.